server/src/test/java/org/elasticsearch/cluster/metadata/WildcardExpressionResolverTests.java

```diff
@@ -215,8 +215,10 @@ public void testResolveAliases() {
 
     public void testResolveDataStreams() {
         String dataStreamName = "foo_logs";
-        IndexMetadata firstBackingIndexMetadata = createBackingIndex(dataStreamName, 1).build();
-        IndexMetadata secondBackingIndexMetadata = createBackingIndex(dataStreamName, 2).build();
+        long epochMillis = randomLongBetween(1580536800000L, 1583042400000L);
+        String dateString = DataStream.DATE_FORMATTER.formatMillis(epochMillis);
```

Comment:
maybe use the `getDefaultBackingIndexName(...)` method that accepts a timestamp?